files/scripts/swss.sh

```diff
@@ -339,6 +339,13 @@ start() {
         $SONIC_DB_CLI APPL_STATE_DB FLUSHDB
         clean_up_chassis_db_tables
         rm -rf /tmp/cache
+        MEDIA_SETTINGS="/usr/share/sonic/device/$PLATFORM/media_settings.json"
+        if [ -f $MEDIA_SETTINGS ]; then
+            # Need to restart PMON on media_settings.json skus due to
+            # https://github.com/sonic-net/sonic-buildimage/issues/21902
+            debug "Restarting pmon service..."
+            /bin/systemctl restart pmon
```

Comment:
@arista-nwolfe Why do we need to restart entire PMON docker? Instead restart only Xcvrd?

Reply:
@arista-nwolfe wont' this cause pmon to restart twice on a platform where pmon and swss come up in independent order ? Have you tested the cold reboot scenario?

Reply:
Restarting xcvrd in latest commit